packages/react-components/src/components/StreamMedia.tsx

```diff
@@ -1,12 +1,20 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { mergeStyles } from '@fluentui/react';
+import { mergeStyles, Spinner } from '@fluentui/react';
 import React, { useEffect, useRef, useState } from 'react';
-import { invertedVideoInPipStyle, mediaContainer } from './styles/StreamMedia.styles';
+import {
+  invertedVideoInPipStyle,
+  mediaContainer,
+  container,
+  loadingSpinnerContainer,
+  loadSpinnerStyles
+} from './styles/StreamMedia.styles';
 import { useTheme } from '../theming';
 import { BaseCustomStyles } from '../types';
 
+type LoadingState = 'loading' | 'none';
```

Comment:
note: this will need exported publicly
Then update both the stable and beta api files with `rush build-api:all-flavors`